hbase-client/src/main/java/org/apache/hadoop/hbase/client/ShortCircuitMasterConnection.java

```diff
@@ -1,716 +0,0 @@
-/**
```

Comment:
Short-circuit works still in async world?

Reply:
No. As for async client, we will use the protobuf `Interface` instead of `BlockingInterface` at client side, but at server side, we only implement `BlockingInterface`. We can introduce the short circuit connection back after we change to implement `Interface` at server side.

Reply:
These TODOs are written up somewhere? In release notes on the issue maybe?

Reply:
We can add a fat release note for HBASE-21512 to mention all these changes.